celements-xwiki-core/src/main/java/com/xpn/xwiki/user/impl/xwiki/XWikiRightServiceImpl.java

```diff
@@ -70,40 +70,62 @@ public class XWikiRightServiceImpl implements XWikiRightService {
       EntityType.SPACE);
 
   private static final Map<String, String> ACTION_MAP = ImmutableMap.<String, String>builder()
-      .put("login", "login")
-      .put("logout", "login")
-      .put("loginerror", "login")
-      .put("loginsubmit", "login")
-      .put("view", "view")
-      .put("edit", "edit")
-      .put("viewrev", "view")
-      .put("get", "view")
-      .put("downloadrev", "view")
-      .put("plain", "view")
-      .put("raw", "view")
+      .put("admin", "admin")
+      .put("ajax", "view")
+      .put("app", "view")
       .put("attach", "view")
+      .put("cancel", "edit")
       .put("charting", "view")
-      .put("skin", "view")
-      .put("download", "view")
-      .put("dot", "view")
-      .put("svg", "view")
-      .put("pdf", "view")
+      .put("commentadd", "comment")
+      .put("create", "edit")
+      .put("delattachment", "edit")
       .put("delete", "delete")
       .put("deleteversions", "admin")
-      .put("undelete", "undelete")
-      .put("reset", "delete")
-      .put("commentadd", "comment")
-      .put("register", "register")
-      .put("redirect", "view")
-      .put("admin", "admin")
+      .put("download", "view")
+      .put("downloadrev", "view")
```

Comment:
```suggestion
      .put("downloadrev", "edit")
```
older versions might be faulty and as that should be talken "offline" for public eyes.